src/__tests__/merge.test.ts

```diff
@@ -178,7 +270,7 @@ describe('mergeOperations()', () => {
         path: '/a',
         responses: [{ code: '200', contents: [], headers: [] }, { code: '400' }, { code: '500' }],
         servers: [],
-        request: { headers: [] },
+        request: { headers: undefined, path: undefined, query: undefined, body: undefined },
```

Comment:
It is because of the notation here: https://github.com/stoplightio/http-spec/pull/110/files#diff-108eac4f51e2b99cad5e9affa181be19R129.

I like the notation because I don't need to create an object and the conditionally add properties to it. But the consequence is that all of them exist in the final object and are eventually set to undefined.

Do you have strong opinion on that?

Reply:
Btw. I just learned that Jest includes presence of fields set to undefined in the diffs. But it does not fail when a field is =undefined vs. is not present in the object.

I just removed `undefined`'s from most of the postman tests.